### DISCUSSION TITLE
Proper names for DSOs, sources, authorities, name collisions?

### DISCUSSION THREAD

**Discussion by vnp85** [Stellarium/stellarium#2134] [Q&A] [answered]
Hi,

I've been wondering about your sources for proper names of deep sky objects. Since a few versions back, the Triple Cave Nebula popped up, which kinda surprised me. Although it has mentions on the internet, I am only familiar with Barnard's E. Since Stellarium is very popular, and kudos, it may accidentally become a de facto naming authority (if it hasn't already): if Stellarium says that's the name, then that is the name. Also, I don't really use stellarium for this kind of viewing, I go Aladin Lite, and see below, but others do, and talking to them, funny things emerge.

Myself, both a programmer, astrophotographer and a philologist (hence my interest), I've been developing an astrophoto planning tool for the past five years (opensourcing it is a WIP), so I ran into the problem of traditional and less traditional names. And while I compiled the names I found on the internet, cause I want them to be resolved in my program, I used especially somewhat old and/or somewhat authoritative sources (like, if Sir Patrick Moore says Cave nebula, then it is SH2-155, or maybe Cederblad 201 as Simbad resolves the Cave Nebula, but not as randomdude calls something on their instagram feed), I find that nowadays it's the wild west with click bait titles all over the place, and more traditional names getting replaced, like the Rosette nebula becoming Skull for some weird Adam-gives-a-new-name-reason. 

I also know that the IAU is of no help, as it regulates catalog designations, not proper names. http://cdsweb.u-strasbg.fr/Dic/iau-spec.html and going by NASA and its supermoon click baits is also counter productive, a few examples aside, like the pillars of creation. Simbad is also of little help, as only a few of the popular names made it into their resolver, probably for this very reason.

I used this repo as a starting point, for my program, but I extended it with several catalogs as I intend to use it offline, https://github.com/astronexus/HYG-Database and also with Hungarian and Romanian traditional names, like Örvény-köd for Whirlpool nebula.

So, do you have a methodology, a secret, a something you could share?

Thank you,
P

**Comment by alex-w** [comment, marked as answer]
See list of sources for DSO names in file https://github.com/Stellarium/stellarium/blob/master/nebulae/default/names.dat (lines 10-84) - this is books, magazines, various databases, atlases, etc. If some name has many sources then if going first in list of aliases. Right now we did not introduced the quality levels for sources, because we still in discussion for "parameters of quality". The last idea for names - just allow users display names from marked by user list of sources.

**Reply by vnp85** [reply]
really helpful, thank you -- maybe I'm not as good at googling as I think I am :P 

Although entries like PK   080-10.1        _("Methuselah Nebula") # APOD ... I mean, APOD as a source is at least controversial, given their history to go clickbait, and even retract fake issues. 

However, the way you document your sources is admirable.

**Reply by gzotti** [reply]
Yes, the need for documenting obscure sources came up some time ago for these very reasons, so that these dubious sources can at least be filtered away in a future step. I fully agree that randomdude's instagram or "Adam's" opinion or some subtitle-like designation given to an APOD image are no qualified sources for names. I also agree that well-established traditional early-20th century (or even older) names like Rosetta or Swan/Omega need no fancy replacements.
